lib/screen/interactive_postcard/postcard_explain.dart

```diff
@@ -26,21 +28,42 @@ class _PostcardExplainState extends State<PostcardExplain> {
   bool _isLastPage = false;
   final VideoPlayerController _controller =
       VideoPlayerController.asset("assets/videos/postcard_explain.mp4");
+  final VideoPlayerController _colouringController =
+      VideoPlayerController.asset("assets/videos/colouring_video.mp4");
```

Comment:
Suggestion: I think we can move these videos to external (github) to reduce the app binary size.
But we can do it in the refactor stories later (maybe in the next cycle), please take a note.